_includes/configuration/iframe-aria-text.md

```diff
@@ -0,0 +1,18 @@
+## `iframe_aria_text`
+
+{{ site.requires_5_9v }}
+
+This option allows you to set the editor iframe title attribute.
```

Comment:
`you` shouldn't be used here, see https://www.tiny.cloud/docs/configure/contributing-docs/#pronouns. We also need to clarify this only works in classic/iframe mode and not other modes such as inline mode.